runner/buckets.go

```diff
@@ -0,0 +1,8 @@
+package runner
+
+var customBuckets = map[string][]float64{
+	"gateway.response_time": {
+		0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10, 60,
+		300, /* 5 mins */
```

Comment:
We have a timeout of 10sec, thus 300 might not be really required

Reply:
Why do we have the histogram showing requests taking much more than that then? :thinking: 
In the PR description I posted a screenshot depicting that.

Reply:
Http server is configured with a [WriteTimeout](https://github.com/rudderlabs/rudder-server/blob/master/gateway/gateway.go#L1387) of 10sec.
Even if we don't honour this at the handler and take more than 10sec to prepare a response for a client's request, my understanding is that the client will never receive this response, but will receive a proxy error instead

Reply:
If that is the case then yeah, anything >10s is actionable from a metrics perspective. I can remove 300 but I'll leave 60 just in case because the `WriteTimeout` is configurable after all, perhaps some customer wants to increase the timeout. In that case at least we will have it covered until one minute.